test/prime_db_schema_client.py

```diff
@@ -2,7 +2,7 @@
 from layman import settings, app
 from layman.common.prime_db_schema import publications, workspaces, users
 
-latest_oauth2_sub = 0
+LATEST_OAUTH2_SUB = 0
```

Comment:
As it is variable and not constant, I prefer lower case and `# pylint: disable=invalid-name`